packages/core/src/App/Containers/Layout/header/header.jsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
+import { connect } from 'Stores/connect';
+import { useLocation } from 'react-router-dom';
```

Comment:
```suggestion
import { useLocation } from 'react-router-dom';
import { connect } from 'Stores/connect';
```

Reply:
there's documentation mentioning about the order of the imports but sadly I couldn't find it 😂 but basically what you should always follow this order: 
1. import React
2. import Third-Party libraries
3. whatever packages from deriv
bla bla bla